test/object_store/test_backup.py

```diff
@@ -148,6 +148,30 @@ async def test_backup_to_non_existent_bucket(manager: ManagerClient, s3_server):
     assert status['state'] == 'failed'
 
 
+async def test_backup_to_non_existent_endpoint(manager: ManagerClient, s3_server):
```

Comment:
Why is this test added by a patch titled "backup: Optimize S3 throughput with shard-based upload". Apparently it tests something else

Reply:
Moved to a new commit